.github/workflows/push.yml

```diff
@@ -0,0 +1,63 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+    - v*
```

Comment:
I think the tags we usually push on release don't have a `v` in front of them, so if this is meant to trigger on that it won't work.